From 6cdadc8fc29c8e26c2f9760975e80bb020fe5ef0 Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Fri, 11 May 2012 15:51:10 +0530
Subject: [PATCH 0962/1092] ARM: OMAP2+: am335x: remove in-correct comment

am33xx_evmid_fillup() is called by many more boards (BB, EVM-SK) not
just IA. So remove in-correct comment

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/devices.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/mach-omap2/devices.c b/arch/arm/mach-omap2/devices.c
index e7710a4..e125b18 100644
--- a/arch/arm/mach-omap2/devices.c
+++ b/arch/arm/mach-omap2/devices.c
@@ -1300,9 +1300,6 @@ static unsigned int   am33xx_evmid = GEN_PURP_EVM;
 * @evmid - evm id which needs to be configured
 *
 * This function is called to configure board evm id.
-* IA Motor Control EVM needs special setting of MAC PHY Id.
-* This function is called when IA Motor Control EVM is detected
-* during boot-up.
 */
 void am33xx_evmid_fillup(unsigned int evmid)
 {
-- 
1.7.11.2

